src/test/app/Offer_test.cpp

```diff
@@ -5081,6 +5081,157 @@ class Offer_test : public beast::unit_test::suite
         pass();
     }
 
+    void
+    testFillOrKill(FeatureBitset features)
```

Comment:
Thanks very much for adding this test.  I especially like the way it demonstrates identical behavior before flowCross and after flowCross but with your fix.  Nicely done!